utam-compiler/src/main/java/utam/compiler/grammar/UtamRootDescription.java

```diff
@@ -87,15 +84,20 @@ static UtamRootDescription processRootDescriptionNode(JsonNode descriptionNode)
   /**
    * Get description as list of strings to wrap into javadoc format
    *
+   * @param version                page objects with version number
+   * @param sourceFileRelativePath relative path with Json
    * @return list of strings
    */
-  List<String> getDescription() {
+  List<String> getDescription(String version, String sourceFileRelativePath) {
     List<String> descriptionLines = new ArrayList<>(text);
+    if(sourceFileRelativePath != null) {
+      String addComma = text.isEmpty()? "" : ", ";
+      descriptionLines.add(String.format("%screated from JSON %s", addComma, sourceFileRelativePath));
+    }
     // add line @author team_name
     descriptionLines.add(String.format("@author %s", (author == null ? "UTAM" : author)));
     // add line @version with timestamp
-    descriptionLines.add(String.format("@version %s",
-        LocalDateTime.now().format(DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss"))));
+    descriptionLines.add(String.format("@since %s", version));
```

Comment:
I think we still want this to be `@version`. We can figure out how to add `@since` some other time.

Reply:
addressed